tests/functional/object/mcg/test_pv_pool.py

```diff
@@ -344,3 +348,37 @@ def test_ephemeral_for_pv_bs(self, backingstore_factory):
         assert (
             "test_object" in search_output
         ), "Dummy data was not found in the node ephemeral storage!"
+
+    @fips_required
+    @tier2
+    @bugzilla("2247731")
+    @polarion_id()
+    def test_pvpool_bs_in_fips(self, backingstore_factory):
+        """
+        Create PV pool based backingstore and make sure the backingstore doesn't
+        goto Rejected phase on noobaa-core pod restarts
+
+        """
+        # create pv-pool backingstore
+        pv_backingstore = backingstore_factory(
+            "OC",
+            {"pv": [(1, MIN_PV_BACKINGSTORE_SIZE_IN_GB, CEPHBLOCKPOOL_SC)]},
+        )[0]
+
+        # restart noobaa-core pod
+        get_noobaa_core_pod().delete()
+
+        # wait for about 10 mins to check if
+        # the backingstore has reached Rejected state
+        time.sleep(10)
```

Comment:
shouldn't this be 600 if it is 10 minutes?
Also, can you pls add a log.info saying the test is waiting for backingstore to be ready.

Reply:
10 minutes wait is done in the code below this, where we wait for backingstore state. This 10 seconds wait was given for testing purpose.